docs/package.json

```diff
@@ -18,13 +18,13 @@
   },
   "dependencies": {
     "@docusaurus/core": "^2.0.0-alpha.61",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.61",
+    "@docusaurus/plugin-client-redirects": "^2.0.0-alpha.61",
+    "@docusaurus/plugin-content-docs": "^2.0.0-alpha.61",
+    "@docusaurus/plugin-sitemap": "^2.0.0-alpha.61",
+    "@docusaurus/theme-classic": "^2.0.0-alpha.61",
     "@mdx-js/react": "^1.5.8",
-    "clsx": "^1.1.1",
     "react": "^16.8.4",
-    "react-dom": "^16.8.4",
-    "remark-collapse": "^0.1.2",
-    "remark-sources": "^1.0.3"
+    "react-dom": "^16.8.4"
   },
```

Comment:
I see what you mean now about the dependencies! I shouldnt have added them before, it just makes it more work to try to get the deployment working. We will almost certainly use them, which is why i've left the comments in the docusaurus config to remind me where to put them back in, but i'll update those on the content branch: https://github.com/RasaHQ/rasa-sdk/pull/254/files

Sorry for the confusion on the last one 😄